homepage/tests.py

```diff
@@ -26,4 +27,27 @@ def test_valid_form(self):
 
     def test_non_valid_form(self):
         form = Blog()
-        self.assertFalse(form.is_valid())
\ No newline at end of file
+        self.assertFalse(form.is_valid())
+    def test_tittle_gt_body(self):
+        form = Blog({'post_tittle':'length of title greater than length of body','post_content':'less than tittle'})
+        self.assertFalse(form.is_valid())
+    def test_tittle_ls_body(self):
+        form = Blog({'post_tittle':'length of tittle','post_content':'less than tittle.now content is graeter than the length of the body'})
+        self.assertTrue(form.is_valid())
+    def test_char_len_lt_10(self):
+        form = Blog({'post_tittle':'leng','post_content':'less th'})
+        self.assertFalse(form.is_valid())
+class Editlinl(TestCase):
+    def setUp(self):
+        self.post = Post.objects.create(post_tittle='testing purpose', post_date=timezone.now(),
```

Comment:
Have consistent indentation, and give `post_date` its own line like other parameters